@autoantwort ,Thanks for your pr, could you please take the error log:
````

CMake Error at /agent/_work/1/s/scripts/buildsystems/vcpkg.cmake:540 (_add_executable):
  Target "kioexec" links to target "LibMount::LibMount" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  src/kioexec/CMakeLists.txt:28 (add_executable)


CMake Error at /agent/_work/1/s/scripts/buildsystems/vcpkg.cmake:578 (_add_library):
  Target "kio_trash" links to target "LibMount::LibMount" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  src/ioslaves/trash/CMakeLists.txt:30 (add_library)


CMake Error at /agent/_work/1/s/scripts/buildsystems/vcpkg.cmake:578 (_add_library):
  Target "kcm_trash" links to target "LibMount::LibMount" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  src/ioslaves/trash/CMakeLists.txt:95 (add_library)
````

--- a/ports/fontconfig/portfile.cmake
+++ b/ports/fontconfig/portfile.cmake
@@ -38,10 +38,7 @@ endif()
 # Make path to cache in fonts.conf relative
 set(_file "${CURRENT_PACKAGES_DIR}/etc/fonts/fonts.conf")
 if(EXISTS "${_file}")
-    file(READ "${_file}" _contents)
-    string(REPLACE "${CURRENT_INSTALLED_DIR}/var/cache/fontconfig" "./../../var/cache/fontconfig" _contents "${_contents}")
-    string(REPLACE "/var" "/../var" _contents "${_contents}")
-    file(WRITE "${_file}" "${_contents}")
+    vcpkg_replace_string("${_file}" "${CURRENT_PACKAGES_DIR}/var/cache/fontconfig" "./../../var/cache/fontconfig")
 endif()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/var"

--- a/ports/fontconfig/vcpkg.json
+++ b/ports/fontconfig/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "fontconfig",
   "version": "2.13.94",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Library for configuring and customizing font access.",
   "homepage": "https://www.freedesktop.org/wiki/Software/fontconfig",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2218,7 +2218,7 @@
     },
     "fontconfig": {
       "baseline": "2.13.94",
-      "port-version": 1
+      "port-version": 2
     },
     "foonathan-memory": {
       "baseline": "2019-07-21",

--- a/versions/f-/fontconfig.json
+++ b/versions/f-/fontconfig.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "bc9133bf9d7f323757e7de9992ae6dbac0a92f00",
+      "version": "2.13.94",
+      "port-version": 2
+    },
+    {
       "git-tree": "606b2d7ca0ba11e29552311100b66d8c5052faeb",
       "version": "2.13.94",
       "port-version": 1

